packages/components/src/TransactionButton.tsx

```diff
@@ -72,14 +73,29 @@ export class TransactionButtonNoModal extends React.Component<TransactionButtonP
   }
 
   public render(): JSX.Element {
+    let StyledButton;
```

Comment:
Ok, I ended up taking both of your suggestions @walfly.  Thx, it def looks more readable.